README.md

```diff
@@ -9,13 +9,31 @@ the [Elixium Network](https://www.elixiumnetwork.org)
 
 ## Usage
 
+# Standard Usage
 ```elixir
 {:ok, ref} = WaspVM.start() # Start WaspVM
 WaspVM.load_file(ref, "path/to/wasm/file.wasm") # Load a module
 WaspVM.execute(ref, "some_exported_function") # Call a function
 # => {:ok, total_gas_cost, :function_return_value}
 ```
 
+# Options
+For Gas Limit use:
+``` elixir
```

Comment:
These options should also be added to the `execute/4` function docs so they show up on the hexdocs

Reply:
Adding Now